<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_forgot">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>371</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="forgot_title">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Forgot Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="em_us_forgot_linedit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Email or Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>80</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter your email or username to see</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>34</x>
       <y>102</y>
       <width>291</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>your password.</string>
     </property>
    </widget>
    <widget class="QLabel" name="kapcha">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>captcha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_kapch_lineedit">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter captcha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="forgot_password_btn">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>240</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>my Password</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>499</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
